lua/DCS-gRPC/methods/net.lua

```diff
@@ -41,3 +41,14 @@ GRPC.methods.getPlayers = function()
 
     return GRPC.success({players = players})
 end
+
+GRPC.methods.forcePlayerSlot = function(params)
```

Comment:
Is there a way to force back to spectator via this API? If so, what is the incantation and can we document it.

Reply:
Yep! We can force people into very specific Blue/Red slots.